include/session.php

```diff
@@ -40,12 +40,12 @@ function ref_session_write ($id,$data) {
 	if($session_exists) {
 		$r = q("UPDATE `session`
 				SET `data` = '%s'
-				WHERE `data` != '%s' AND `sid` = '%s'",
+				WHERE `sid` = '%s' AND `data` != '%s'",
 				dbesc($data), dbesc($data), dbesc($id));
```

Comment:
I'm unsure if the changed order really is improving performance. Do you have reliable data about it?